### DISCUSSION TITLE
Calendar disappeared for user - but still visible for all others

### DISCUSSION THREAD

**Discussion by Schorse3333** [nextcloud/all-in-one#4764] [Questions] [answered]
Hardware/Software Info: I am using Nextcloud AIO (v8.2.1) on a Lenovo ThinkCentre M75q Gen 2 with 16GB RAM running Ubuntu 22.04.02 LTS

**Problem:**
I recently received an email from one of our users, that her calendar had disappeared (1) and that every new calendar entry is now associated to the calendar of another user (2).

While case (2) is easily to explain - she has editing rights in this calendar, so Nextcloud just simply picks the next valid calendar - I am struggling to find the root cause for case (1).

If I log in myself, I can see her calendar (even with her profile icon next to it) in my calendar view, as it was shared previously with a certain user group. However, if I use "Imitation" to check her user, I cannot see this calendar at all. It is simply not there and is also not available in the dropdown when creating a new entry.

Here is what I see (Calendar visible):
![image](https://github.com/nextcloud/all-in-one/assets/83947844/c2760b02-70d0-473d-a1fa-bca2aec70825)

Here is what the user sees (Calendar not visible):
![image](https://github.com/nextcloud/all-in-one/assets/83947844/5e9c4c25-b829-4f21-931e-630cc9ddeb5c)

**First thoughts:**

To me this means, that the calendar seems to be there but no longer associated with the user.

Of course I can create a backup ICS file of the calendar easily from my admin user account and have her create a new one in her profile. But the former one, which was shared with a wider group, will still remain visible and cannot be removed.

Any thoughts on how to approach this or how to start troubleshooting? I am to a certain extent familiar with Ubuntu but not en expert. The same applies for docker. So I would appreciate your help on the "how" in case you need some further information.+

Thanks, Schorse333

**Comment by szaimen** [comment, marked as answer]
Hi, please ask this at https://help.nextcloud.com
